src/game.rs

```diff
@@ -173,7 +173,7 @@ impl Game {
     }
 
     fn is_victory(&self) -> bool {
-        self.foundations.iter().all(|f| f.is_full())
+        self.cascades.iter().all(|c| c.has_all_alternating())
```

Comment:
This clearly breaks the normal way to finish the game if the last card you're stacking is in a cell.